themes-default/slim/package.json

```diff
@@ -113,18 +125,27 @@
     "esnext": true
   },
   "ava": {
+    "require": [
+      "./test/helpers/setup.js"
+    ],
     "files": [
-      "vue/test/**/*.js"
+      "test/specs/**/*.spec.js"
     ],
-    "source": [
-      "vue/src/**/*.{js,jsx}",
-      "!dist/**/*"
+    "sources": [
+      "views/**/*.vue",
+      "static/js/**/*.js"
```

Comment:
Sources are incorrect, as the `views` folder doesn't/won't hold any `.vue` files according to #4374.
For now you can probably use the following, if that works...
```js
"sources": [
  "static/js/**/*.{js,vue}",
  "views/vue-components/*.mako"
]
```